.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build & test
+
+on:
+  pull_request:
+  push:
+    branches: [ main, release/* ]
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK
+      uses: actions/setup-java@v2
+      with:
+        java-version: 8
```

Comment:
Why did you pick Java 8?

Reply:
It probably it doesn't matter that much. Just figured we'd push for 11 or 17 if we can.
@mbayerPK / @PKJosh Is Unpacker still driving a java 8 baseline?

Reply:
yeah unpacker runs with java 17 but compiles/builds for java 8 (because DSM/DME requires it)

Reply:
my third party projects run on java 11 but compile/build for java 8 btw (so you dont have to enforce necessarily setting java to 8 here)

see https://github.com/pkware/AttributeCachingFileSystem/blob/main/file-attribute-caching/build.gradle.kts and the kotlin block using javaToolChains and https://github.com/pkware/AttributeCachingFileSystem/blob/main/build.gradle.kts using the jvm target to hit 1.8 there (ie java 8)